test/WebJobs.Script.Tests/WebJobs.Script.Tests.csproj

```diff
@@ -56,7 +56,6 @@
     <PackageReference Include="Microsoft.ApplicationInsights.WindowsServer" Version="2.20.0" />
     <PackageReference Include="Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel" Version="2.20.0" />
     <PackageReference Include="Microsoft.Extensions.Logging.ApplicationInsights" Version="2.20.0" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.4.0" />
```

Comment:
Getting a build error because further below there's a duplicate line for this package referencing version 16.9.4.

Reply:
Lilian had this issue fixed in dev branch in [this PR](https://github.com/Azure/azure-functions-host/pull/8579). Not sure how it got checked in to source control without build failing.